lib/rubocop/ast/node.rb

```diff
@@ -22,40 +22,65 @@ class Node < Parser::AST::Node # rubocop:disable Metrics/ClassLength
       include RuboCop::AST::Sexp
       extend NodePattern::Macros
 
+      # @api private
       # <=> isn't included here, because it doesn't return a boolean.
-      COMPARISON_OPERATORS = %i[== === != <= >= > <].freeze
+      COMPARISON_OPERATORS = %i[== === != <= >= > <].to_set.freeze
 
+      # @api private
       TRUTHY_LITERALS = %i[str dstr xstr int float sym dsym array
                            hash regexp true irange erange complex
-                           rational regopt].freeze
-      FALSEY_LITERALS = %i[false nil].freeze
+                           rational regopt].to_set.freeze
+      # @api private
+      FALSEY_LITERALS = %i[false nil].to_set.freeze
+      # @api private
       LITERALS = (TRUTHY_LITERALS + FALSEY_LITERALS).freeze
+      # @api private
       COMPOSITE_LITERALS = %i[dstr xstr dsym array hash irange
-                              erange regexp].freeze
+                              erange regexp].to_set.freeze
+      # @api private
       BASIC_LITERALS = (LITERALS - COMPOSITE_LITERALS).freeze
+      # @api private
       MUTABLE_LITERALS = %i[str dstr xstr array hash
-                            regexp irange erange].freeze
+                            regexp irange erange].to_set.freeze
+      # @api private
       IMMUTABLE_LITERALS = (LITERALS - MUTABLE_LITERALS).freeze
 
+      # @api private
       EQUALS_ASSIGNMENTS = %i[lvasgn ivasgn cvasgn gvasgn
-                              casgn masgn rasgn mrasgn].freeze
-      SHORTHAND_ASSIGNMENTS = %i[op_asgn or_asgn and_asgn].freeze
+                              casgn masgn rasgn mrasgn].to_set.freeze
+      # @api private
+      SHORTHAND_ASSIGNMENTS = %i[op_asgn or_asgn and_asgn].to_set.freeze
+      # @api private
       ASSIGNMENTS = (EQUALS_ASSIGNMENTS + SHORTHAND_ASSIGNMENTS).freeze
```

Comment:
-1
```suggestion
       ASSIGNMENTS = 8```suggestion
       ASSIGNMENTS = (EQUALS_ASSIGNMENTS + SHORTHAND_ASSIGNMENTS).freeze
```
(EQUALS_ASSIGNMENTS + SHORTHAND_ASSIGNMENTS).freeze
```

Reply:
```suggestion
2**@_[]()_**